Backpack/Card/Classes/BPKCard.m

```diff
@@ -65,9 +66,9 @@ - (instancetype)initWithPadded:(BOOL)padded {
 
 - (void)setPadded:(BOOL)padded {
     if (padded) {
-        self.layoutMargins = UIEdgeInsetsMake(BPKSpacingBase, BPKSpacingBase, BPKSpacingBase, BPKSpacingBase);
+        self.innerView.layoutMargins = UIEdgeInsetsMake(BPKSpacingBase, BPKSpacingBase, BPKSpacingBase, BPKSpacingBase);
```

Comment:
I would execpt us to still set the `layoutMaring` on `self` here and have the innerView constrianted to the layout guides of `self`